archivebot.py

```diff
@@ -137,6 +137,23 @@ def handle_query(event, cursor, say):
         limit = 10
 
         s = event["text"].lower()
+        
+        # take into account this example
+        # s = '"from:Fabio Lombardo" this is a test'
```

Comment:
I the fix a lot! We have many names with spaces!!

That said, I think a more standard approach would be to handle `from:"Fabio Lombardo"` rather than `"from:Fabio Lombardo"`.

Any reason why we would prefer the latter?